generators/server/templates/entity/src/test/java/gatling/simulations/EntityGatlingTest.java.ejs

```diff
@@ -103,82 +118,84 @@ class <%= entityClass %>GatlingTest extends Simulation {
         .formParam("password", "admin")
         .formParam("remember-me", "true")
         .formParam("submit", "Login")
-        .check(headerRegex("Set-Cookie", "XSRF-TOKEN=(.*);[\\s]").saveAs("xsrf_token"))).exitHereIfFailed
+        .check(headerRegex("Set-Cookie", "XSRF-TOKEN=(.*);[\\s]").saveAs("xsrf_token"))).exitHereIfFailed()
 <%_ } else if (authenticationTypeOauth2) { _%>
         .get("/oauth2/authorization/oidc")
-        .check(status.is(302))
-        .check(header("Location").saveAs("loginUrl"))).exitHereIfFailed
+        .check(status().is(302))
+        .check(header("Location").saveAs("loginUrl"))).exitHereIfFailed()
         .pause(2)
         .exec(http("Login Redirect")
         .get("${loginUrl}")
-        .silent
+        .silent()
         .headers(keycloakHeaders)
-        .check(css("#kc-form-login", "action").saveAs("kc-form-login"))).exitHereIfFailed
+        .check(css("#kc-form-login", "action").saveAs("kc-form-login"))).exitHereIfFailed()
```

Comment:
Not sure (it looks good according to the docs), but the failing tests look like a missing import

Reply:
Thanks for the hint, static import `io.gatling.javaapi.core.CoreDsl.css` was missing :-)